numpy/lib/npyio.py

```diff
@@ -1088,9 +1088,10 @@ def loadtxt(fname, dtype=float, comments='#', delimiter=None,
         The string used to separate values. For backwards compatibility, byte
         strings will be decoded as 'latin1'. The default is whitespace.
     converters : dict or callable, optional
-        A function to parse all columns strings into the desired value, or
-        a dictionary mapping column number to a parser function.
-        E.g. if column 0 is a date string: ``converters = {0: datestr2num}``.
+        Converter functions that parse string or bytes data from file to the
+        desired value. If `converters` is callable, the function is applied
+        to all columns, else it must be a dict that maps column number to a
+        parser function. See examples for further details.
```

Comment:
```suggestion
        Converter functions to customize value parsing. The convert is passed
        each values unmodified string (or bytes) file and returns the value
        that is stored in the result. If `converters` is callable, the function
        is applied to all columns, else it must be a dict that maps column
        numbers to a parser function. See examples for further details.
```
I am also happy to just merge it, somehow I find it hard to figure out a very clear way to write it.  (I feel you would need more vocabulary, like field to indicate each value before parsing...)

Reply:
Agreed, it's hard to be concise especially given all the extra complication with `encoding="bytes"` re: what the converter functions expect as input :shrug: 

@keithbriggs do either of the two suggested rewordings above make things clearer in your opinion?